bentoml/cli/aws_lambda_template.py

```diff
@@ -0,0 +1,66 @@
+from ruamel.yaml import YAML
+
+from bentoml.utils import Path
+
+AWS_HANDLER_PY_TEMPLATE = """\
+try:
+    import unzip_requirements:
+except ImportError:
+    pass
+
+import json
+
+import {class_name}
+
+bento_service = {class_name}.load()
+
+def {api_name}(event, context):
+    result = bento_service.{api_name}.handle_aws_lambda_event(event)
+
+    return result
+"""
+
+
+def update_serverless_configuration_for_aws(bento_service, output_path, extra_args):
+    yaml = YAML()
+    api = bento_service.get_service_apis()[0]
+    with open(output_path, 'r') as f:
+        content = f.read()
+    serverless_config = yaml.load(content)
+
+    if extra_args.region:
+        serverless_config['provider']['region'] = extra_args.region
+    else:
```

Comment:
could you add more logging here? something like `logger.info('Using AWS region {extra_args.region}')` here can be very helpful to user

Reply:
done